framework/src/media/audio/README.txt

```diff
@@ -0,0 +1,65 @@
+Audio Manager README
```

Comment:
Why don't you use markdown syntax? We are in github system and that uses markdown to display documentation.